api/client/stats.go

```diff
@@ -117,6 +119,48 @@ func (s *containerStats) Collect(cli *DockerCli, streamStats bool) {
 	}
 }
 
+// CalculateTotal adds up all containers' resource statistics
+func (s *stats) CalculateTotal() {
+	// get total memory
+	var totalMemory int64
+	meminfo, err := system.ReadMemInfo()
```

Comment:
Oh, I'm afraid you're right. 
I'll use the `info` api to get the correct memory in the server side.